syntax = "proto3";

package moderation.input;

message InputDataNode {
    int32 entity_id = 1;
    string unique_id = 2;
    int32 object_id = 3;
    string current_status = 4;
    repeated Fields fields = 5;
}

message Fields {
    int32 id = 1;
    string field_name = 2;
    string new_value = 3;
    string old_value = 4;
    int32 parent_id = 5;
    string parent_field_name = 6;
    repeated string list_value = 7;
    repeated MetaInfo meta_info = 8;
}

message MetaInfo {
    string key = 1;
    string value = 2;
}